tasks.py

```diff
@@ -257,7 +257,14 @@ def reset_db_hard(c):
     import os
     from urllib.parse import urlparse
 
-    db_url = os.environ.get("DATABASE_URL", "")
+    db_url_file = os.environ.get("DATABASE_URL_FILE", "")
```

Comment:
I don't think this will ever be used inside of a docker environment that's using secrets, will it?